defaults/main.yml

```
@@ -4,6 +4,7 @@
 sympa_domain: "{{ ansible_fqdn }}"
 sympa_show_default_templates: false
 sympa_template_lists: []
+sympa_font_awesome_version: "6.0.0"
```

```suggestion
sympa_font_awesome_version: "6.6.0"
```

why don't use the latest release?